### ISSUE TITLE
sscan fails to build for win32-x86 against base-3.15.4

### ISSUE THREAD

**Issue by kmpeters** [epics-modules/sscan#3]
I'm using the following:
```
Windows 7 (64-bit)
Visual Studio 2010 Professional & Visual Studio 2015 Community Edition
Make 4.1 (from the EPICS web site)
Strawberry perl 5.20.3001
```
The sscan trunk (edd43ed2e5205be8bc6cce535807460a5268b64e) successfully builds against the released version of base-3.14.12.5 and fails to build against the released version of base-3.15.4 using both VS2010 and VS2015:
```
link -nologo /subsystem:windows /dll -LTCG -incremental:no -opt:ref -release       -out:sscan.dll -implib:sscan.lib        sscanRecord.obj scanparmRecord.obj re
cDynLink.obj req_file.obj saveData_writeXDR.obj writeXDR.obj      D:/synApps-testing/VS15/3.15/base-3.15.4/lib/win32-x86/dbRecStd.lib  D:/synApps-testing/VS15/3
.15/base-3.15.4/lib/win32-x86/dbCore.lib  D:/synApps-testing/VS15/3.15/base-3.15.4/lib/win32-x86/ca.lib  D:/synApps-testing/VS15/3.15/base-3.15.4/lib/win32-x86/
Com.lib
   Creating library sscan.lib and object sscan.exp
recDynLink.obj : error LNK2001: unresolved external symbol __imp__dbNameToAddr@8
sscan.dll : fatal error LNK1120: 1 unresolved externals
D:/synApps-testing/VS15/3.15/base-3.15.4/configure/RULES_BUILD:281: recipe for target 'sscan.dll' failed
make[4]: *** [sscan.dll] Error 1120
make[4]: Leaving directory 'D:/synApps-testing/VS15/3.15/synApps/support/sscan/sscanApp/src/O.win32-x86'
D:/synApps-testing/VS15/3.15/base-3.15.4/configure/RULES_ARCHS:61: recipe for target 'install.win32-x86' failed
make[3]: *** [install.win32-x86] Error 2
make[3]: Leaving directory 'D:/synApps-testing/VS15/3.15/synApps/support/sscan/sscanApp/src'
D:/synApps-testing/VS15/3.15/base-3.15.4/configure/RULES_DIRS:88: recipe for target 'src.install' failed
make[2]: *** [src.install] Error 2
make[2]: Leaving directory 'D:/synApps-testing/VS15/3.15/synApps/support/sscan/sscanApp'
D:/synApps-testing/VS15/3.15/base-3.15.4/configure/RULES_DIRS:88: recipe for target 'sscanApp.install' failed
make[1]: *** [sscanApp.install] Error 2
make[1]: Leaving directory 'D:/synApps-testing/VS15/3.15/synApps/support/sscan'
D:/synApps-testing/VS15/3.15/base-3.15.4/configure/RULES_DIRS:88: recipe for target 'D:/synApps-testing/VS15/3.15/synApps/support/sscan.install' failed
make: *** [D:/synApps-testing/VS15/3.15/synApps/support/sscan.install] Error 2
```
The latest version of the sequencer (2.2.3) is used.

**Comment by kmpeters** [comment]
@timmmooney suggests the fix:

Remove "epicsShareAPI" in declaration of dbNameToAddr in recDynLink.c when base 3.15 is used.

**Comment by kmpeters** [comment]
This is fixed in commit 07cc4d2b06d0413244a55dd55ddece7951695707
